IES Specification Docs/Information Exchange Standard r4.2.0.n3

```diff
@@ -227,21 +216,26 @@ ies:assessed rdfs:domain ies:AssessToBeTrue .
 ies:assessed rdfs:range rdfs:Resource . 
 ies:assessedToBeTheSameAs rdf:type owl:ObjectProperty . 
 ies:assessedToBeTheSameAs sparx:guid "{6A1C6C65-D671-4ea3-9184-044AE2A802CF}" . 
-ies:assessedToBeTheSameAs rdfs:comment "A relationship that asserts two ExchangedItems that may have been previously judged to be different are in fact the same thing. WARNING:  by the same we mean they occupy the same space for the same period of time - i.e. not two different things in the same place at different times, and not the same physical item at two different periods of time. The Identifier and State patterns should do most of what is needed here, and it is extremely rare that this would ever be needed. Do not use unless absolutely necessary. " . 
+ies:assessedToBeTheSameAs rdfs:comment "A relationship that asserts two Things that may have been previously judged to be different are in fact the same thing. WARNING:  by the same we mean they occupy the same space for the same period of time - i.e. not two different things in the same place at different times, and not the same physical item at two different periods of time. The Identifier and State patterns should do most of what is needed here, and it is extremely rare that this would ever be needed. Do not use unless absolutely necessary. " . 
 ies:assessedToBeTheSameAs rdfs:subPropertyOf ies:relationship . 
+ies:Assessment rdf:type rdfs:Class . 
+ies:Assessment sparx:guid "{E4AB33D9-9978-446f-9C39-4F4C41FB3D45}" . 
+ies:Assessment rdfs:comment "An Event where an actor makes a subjective judgement against a thing. This can be a judgement of belief in a thing�s possibility, categorisation or other qualitative aspect.Examples include:- Having HIGH confidence that Anne committed the murder.- Assessing a statement made in an internet article as with being true or false.- Assessing a house to having an energy performance of B- Assessing the odds of England winning the World Cup as 20-1" . 
```

Comment:
```suggestion
ies:Assessment rdfs:comment "An Event where an actor makes a subjective judgement against a thing. This can be a judgement of belief in a thing�s possibility, categorisation or other qualitative aspect.Examples include:- Having HIGH confidence that Anne committed the murder.- Assessing a statement made in an internet article as being true or false.- Assessing a house to having an energy performance of B- Assessing the odds of England winning the World Cup as 20-1" . 
```

Reply:
Changed in commit: [cd246c9](https://github.com/dstl/IES4/pull/9/commits/cd246c9c4655ada94c356e90f7cafd11bb5d2420)